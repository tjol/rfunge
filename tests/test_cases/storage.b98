as 8f1p 20ga-|
             >' 8-|
v:"s, p, ', g look ok"<>:
_@                >0a ^^,
    "ERROR"a0<    <    ^