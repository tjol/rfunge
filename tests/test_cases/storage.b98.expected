s, p, ', g look ok
